apps/emqx_ft/test/emqx_ft_responder_SUITE.erl

```diff
@@ -20,15 +20,19 @@
 -compile(nowarn_export_all).
 
 -include_lib("stdlib/include/assert.hrl").
+-include_lib("common_test/include/ct.hrl").
 
 all() -> emqx_common_test_helpers:all(?MODULE).
 
 init_per_suite(Config) ->
-    ok = emqx_common_test_helpers:start_apps([emqx_ft], emqx_ft_test_helpers:env_handler(Config)),
-    Config.
+    {ok, Apps} = application:ensure_all_started(gproc),
+    {ok, Pid} = emqx_ft_responder_sup:start_link(),
+    true = unlink(Pid),
+    [{suite_apps, Apps}, {sup_pid, Pid} | Config].
 
-end_per_suite(_Config) ->
-    ok = emqx_common_test_helpers:stop_apps([emqx_ft]),
+end_per_suite(Config) ->
+    ok = proc_lib:stop(?config(sup_pid, Config), shutdown, 5000),
```

Comment:
Nit: 
M.b. just `_ = emqx_ft_responder_sup:start_link()` in `init_per_testcase` to avoid start/stop boilerplate

Reply:
Maybe, but I'm a bit wary of such approach, since processes would then exit with non-"normal" reason, which will produce a lot of noise in the ct logs.
```
=ERROR REPORT==== 22-Jun-2023::10:41:19.031613 ===
** Generic server emqx_ft_responder_sup terminating 
** Last message in was {'EXIT',<0.12925.1>,
                               {#Ref<0.1141479013.47185921.144393>,55837,ok,
                                [{emqx_ft_responder_SUITE,t_start_ack}],
                                []}}
** When Server state == {state,
                            {local,emqx_ft_responder_sup},
                            simple_one_for_one,
                            {[responder],
                             #{responder =>
                                   {child,undefined,responder,
                                       {emqx_ft_responder,start_link,[]},
                                       temporary,false,5000,worker,
                                       [emqx_ft_responder]}}},
                            {mapsets,#{}},
                            100,100,[],0,never,emqx_ft_responder_sup,[]}
** Reason for termination ==
** {#Ref<0.1141479013.47185921.144393>,55837,ok,
    [{emqx_ft_responder_SUITE,t_start_ack}],
    []}
```

Those don't show up if tests pass, yet when some of them fail, there might be a lot of "false positives" to go through in the logs.